src/pages/management/configuration-api/index.mdx

```diff
@@ -1890,12 +1890,12 @@ curl -X POST \
     "name": "string_property",
     "owner_client_id": "0805e283233042b37f460ed8fbf22160",
     "type": "string",
-    "default_value": "string_default_value"
+    "default_value": "string_default_value",
```

Comment:
Corrected requests for the Register property method in all versions as there were two mistakes connected with commas.